### PR TITLE
Berserker/Marked One Armour Nerf

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request
Nerfs the marked one armour a bit. The base form has slightly better values than normal berserker armour, the upgraded version is largely the same but with some extra melee padding

## Why It's Good For The Game

Currently Marked One armour is largely the same as Deathsquad armour with an ash hide upgrade.
I have, however, taken an axe to this problem, and severely nerfed the berserker armour and marked one armour by making it a lot less obscenely powerful when upgraded with an ash drake hide.

The armour is now largely comparable to HECK suit and Ash Drake Armour when upgraded, so its less obscene, if still slightly better because the marked one is a challenging boss.

## Proof Of Testing

<details>
<summary>Screenshots/Videos</summary>

</details>

## Changelog

:cl:
balance: Berserker armour's upgrading capability greatly nerfed.
balance: Marked One armour is scaled back from its upgraded values to be much closer to its base berserker armour.
/:cl:

### CODE REVIEW

**Comment by NargaInDisguise** [comment]
Genuinely, what purpose does nerfing bio protection serve at all?

**Comment by YakumoChen** [comment]
It's not airtight, neither is the armour its made from.

Why is it your concern?

**Comment by NargaInDisguise** [comment]
I was asking because it just seemed like an odd change, genuinely was just curiosity.

**Comment by Doom-external** [comment]
Out of curiosity, how does innate anti-magic make it worse armor?

**Comment by YakumoChen** [comment]
> Out of curiosity, how does innate anti-magic make it worse armor?

prevents you from using  any of the magic bullshit you can also find on lavaland, mostly

**Comment by Doom-external** [comment]
> prevents you from using any of the magic bullshit you can also find on lavaland, mostly, such as:
> 
> * Voice of God
> * Magic books (recall, self-immolate)
> * Polymorph

It does however make you immune to heretics and wizards and any other magical antag/weapon. I think the immunity outweighs the other things used.

**Comment by YakumoChen** [comment]
Decided to just remove the antimagic. If you want it still, destroy tendrils for the actual beserker armour ig

**Comment by Razurath** [comment]
> Decided to just remove the antimagic. If you want it still, destroy tendrils for the actual beserker armour ig

Why remove the antimagic when that has always been the base of the armor? With the nerfed armor values and the fact that the armor actually increases your odds of being wounded, do we need to remove the anti-magic? There are 3-4 other sources on lavaland that don't take up my armor slot and don't need you to break tendrils to get anti-magic.

**Comment by Razurath** [comment]
The marked one's armor has always just been a reskined berserker armor, if you're going to take anti magic off then why not just make it its own thing entirely rather then these weird nerfs that apply to one and not the other? Tendrils are EASIER to break for the berserker armor then killing the marked one. This isn't fixing a problem it's encouraging shit like a skilled heretic player killed the marked one, now they can use his armor and have full berserker potential on top of their magic. You don't even need to be a miner to kill the marked one if you have the skills.

**Comment by YakumoChen** [comment]
> Why remove the antimagic when that has always been the base of the armor? With the nerfed armor values and the fact that the armor actually increases your odds of being wounded, do we need to remove the anti-magic? There are 3-4 other sources on lavaland that don't take up my armor slot and don't need you to break tendrils to get anti-magic.

Because you see validhunters using it more often to kill wizards / heretics rather than abusing it.

You're free to use those other sources as well instead, but the Marked one is easiest to use.

> Tendrils are EASIER to break for the berserker armor then killing the marked one.

You can't reliably get this from tendrils, it's a random chance.

>  This isn't fixing a problem it's encouraging shit like a skilled heretic player killed the marked one, now they can use his armor and have full berserker potential on top of their magic. You don't even need to be a miner to kill the marked one if you have the skills.

If a heretic wants to spend a large chunk of their round on lavaland instead of antagging, they're free to do that, sure. They'll still be stuck with the massive wound vulnerability, but okay? This doesn't even beat the actually good armour heretics get.

**Comment by Razurath** [comment]
Not every heretic gets that armor, it's blade path and anyone else is relying on luck to get it. The issue is that it's not the only source of antimagic from lavaland and it also is a drop from a pain in the ass megafauna. If you know what you're doing, or just want to cheese it, you can kill marked one in the first 20 minutes of the shift or less. I think you're going to strip the antimagic then rework it entirely to be something unique and not a downgrade from berserker armor.

You are nerfing:
biological protection
armor values
boosting chance to wounded
removing antimagic

Why is this still megafauna loot considering the nerfs being applied to the sword as well? Everything about the drops from the Marked One are increasingly just making it a sort of status symbol that the crew fears, as he is the hardest megafauna to fight. I agree nerfs are needed, but I think it's getting a little heavy handed. Mobs on lavaland can also cause wounds if you don't have armor to block it. This is becoming increasingly just a cosmetic reward for killing the marked one when originally it was meant to be an actual prize. I get the want to push back against miners fighting antags, but if only 1 job gets anti-magic then they need have the role regarding those antags revised to be allowed to use it, as currently the chaplain is a civilian who happens to be magic proof if they have their null rod and many of those have also been nerfed in terms of combat potential.

**Comment by YakumoChen** [comment]
> You are nerfing:
> biological protection
> armor values
> boosting chance to wounded
> removing antimagic

All of these are fine to nerf, the armour is incredibly powerful currently, consistent to get for anyone who has found a way to beat (or cheese, as you said) the marked one and the values are literally Deathsquad level at current. The wounding is entirely the suggestion of Azalea and doesn't even effect lavaland fighting. Now it's just slightly better than the berserker armour, without the antimagic (which can be good or bad depending on what you're doing anyway)

**Comment by Razurath** [comment]
> > You are nerfing:
> > biological protection
> > armor values
> > boosting chance to wounded
> > removing antimagic
> 
> All of these are fine to nerf, the armour is incredibly powerful currently, consistent to get for anyone who has found a way to beat (or cheese, as you said) the marked one and the values are literally Deathsquad level at current. The wounding is entirely the suggestion of Azalea and doesn't even effect lavaland fighting. Now it's just slightly better than the berserker armour, without the antimagic (which can be good or bad depending on what you're doing anyway)
> 
> I haven't really even nerfed the Marked One armour so much as I've just nerfed the base Berserker armour because upgrading it makes it incredibly strong. Now its on par (slightly better) than every other armour on lavaland.

How is it better then the base berserker armor if the base armor has antimagic still and the marked one armor does not?

**Comment by YakumoChen** [comment]
If antimagic and wound resistance (validhunting on station) is your goal then yes, the berserker armour is better, but you're not going to consistently find it every round regardless, you still have to destroy tendrils and hope you find one.

**Comment by Razurath** [comment]
Okay but you did nothing to make the marked one armor better or even equal to the berserker armor. Even possible uses on station, it's just a downgrade of an armor from what is a difficult fight. Also you don't need to kill tendrils to get chests, you can bomb fish in lava or let lobsters fish up chests. You under estimate how easy it is to get berserker armor vs marked one. But it's merged and you clearly don't intend to change your mind.

**Comment by YakumoChen** [comment]
*Test merged

Pretty sure the lava only has one chest max you can fish out of it.

I don't mind returning the wound resistance, but it was Azalea's idea and they're wearing it half the time anyway

**Comment by Razurath** [comment]
I have gotten sixteen chests using gibtonite in lava in about 30 minutes as an ash walker mining the gibtonite out and using a raptor to move it.

**Comment by YakumoChen** [comment]
> I have gotten sixteen chests using gibtonite in lava in about 30 minutes as an ash walker mining the gibtonite out and using a raptor to move it.

I believe this is getting fixed if true and not done already, it's only intended one chest can be fished out.

**Comment by YakumoChen** [comment]
https://github.com/tgstation/tgstation/pull/86125

Lo and behold, its true

**Comment by YakumoChen** [comment]
Lobsters (without sapience) fishing up chest is similarly a bug fixed upstream

File: code/modules/mining/lavaland/tendril_loot.dm [hunk 1]
```diff
@@ -694,8 +694,8 @@
 
 /datum/armor/drake_empowerment
 	melee = 35
-	laser = 30
-	energy = 20
+	laser = 10 //BUBBER EDIT: Removes overpowered empowerment
+//	energy = 20 //BUBBER EDIT: Removes overpowered empowerment
 	bomb = 20
```
[the remark this answers is not in the document]
Comment (YakumoChen) [reply code/modules/mining/lavaland/tendril_loot.dm:699]: moved it to the other file I changed, not sure if that works

**Comment by YakumoChen** [comment]
re-added the antimagic and removed the negative wound armour

**Comment by Majkl-J** [comment]
Looks good to me

**Comment by Razurath** [comment]
Sad the suit and boss have been removed now.

**Comment by YakumoChen** [comment]
still is a nerf to berserker armour so its fine, especially if the boss returns later.

**Comment by Majkl-J** [comment]
> Sad the suit and boss have been removed now.

They both still exist it's just that the boss needs to be reworked into a basicmob because of the buggy AI and the fact that simplemobs are bound to be deprecated on TG.

**Pull request merged**